package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "simple-json-db",
-  "version": "1.2.2",
+  "version": "1.3.0",
```

Please don't bump the package's version yet.

```suggestion
  "version": "1.2.2",
```